[0KRunning with gitlab-runner 18.3.0 (9ba718cd)[0;m
[0K  on devops ET6Pgvf4s, system ID: r_IcypqJYchyXa[0;m
section_start:1756743161:prepare_executor[0K[0K[36;1mPreparing the "docker" executor[0;m[0;m
[0KUsing Docker executor with image docker:latest ...[0;m
[0KStarting service docker:dind...[0;m
[0KUsing effective pull policy of [always] for container docker:dind[0;m
[0KPulling docker image docker:dind ...[0;m
[0KUsing docker image sha256:b2a8bf985f76be7345c0c827c568db287ccb4dfad919a4693488224e3644fb0c for docker:dind with digest docker@sha256:c0872aae4791ff427e6eda52769afa04f17b5cf756f8267e0d52774c99d5c9de ...[0;m
[0KWaiting for services to be up and running (timeout 30 seconds)...[0;m

[0;33m*** WARNING:[0;m Service runner-et6pgvf4s-project-1-concurrent-0-56997143daec50dc-docker-0 probably didn't start properly.

Health check error:
service "runner-et6pgvf4s-project-1-concurrent-0-56997143daec50dc-docker-0-wait-for-service" timeout

Health check container logs:
2025-09-01T16:12:47.739250491Z waiting for TCP connection to 172.17.0.3 on [2375 2376]...
2025-09-01T16:12:47.739366351Z dialing 172.17.0.3:2376...
2025-09-01T16:12:47.739490707Z dialing 172.17.0.3:2375...
2025-09-01T16:12:48.739945754Z dialing 172.17.0.3:2376...
2025-09-01T16:12:48.740032909Z dialing 172.17.0.3:2375...

Service container logs:
2025-09-01T16:12:48.192111478Z Certificate request self-signature ok
2025-09-01T16:12:48.192174948Z subject=CN=docker:dind server
2025-09-01T16:12:48.207512140Z /certs/server/cert.pem: OK
2025-09-01T16:12:48.566108810Z Certificate request self-signature ok
2025-09-01T16:12:48.566133687Z subject=CN=docker:dind client
2025-09-01T16:12:48.581934608Z /certs/client/cert.pem: OK
2025-09-01T16:12:48.588651196Z ip: can't find device 'nf_tables'
2025-09-01T16:12:48.590385888Z modprobe: can't change directory to '/lib/modules': No such file or directory
2025-09-01T16:12:48.592680503Z ip: can't find device 'ip_tables'
2025-09-01T16:12:48.594055018Z modprobe: can't change directory to '/lib/modules': No such file or directory
2025-09-01T16:12:48.595434704Z ip: can't find device 'ip6_tables'
2025-09-01T16:12:48.596976165Z modprobe: can't change directory to '/lib/modules': No such file or directory
2025-09-01T16:12:48.599454242Z iptables v1.8.11 (nf_tables)
2025-09-01T16:12:48.602950902Z mount: permission denied (are you root?)

[0;33m*********[0;m

[0KUsing effective pull policy of [always] for container docker:latest[0;m
[0KPulling docker image docker:latest ...[0;m
[0KUsing docker image sha256:b2a8bf985f76be7345c0c827c568db287ccb4dfad919a4693488224e3644fb0c for docker:latest with digest docker@sha256:c0872aae4791ff427e6eda52769afa04f17b5cf756f8267e0d52774c99d5c9de ...[0;m
section_end:1756743201:prepare_executor[0Ksection_start:1756743201:prepare_script[0K[0K[36;1mPreparing environment[0;m[0;m
[0KUsing effective pull policy of [always] for container sha256:c909dca3aaaed2a2f10fa96499aee4c9f51fb469c8e8ec2a96059c2172d6b72c[0;m
Running on runner-et6pgvf4s-project-1-concurrent-0 via 01d950ce02bd...
section_end:1756743202:prepare_script[0Ksection_start:1756743202:get_sources[0K[0K[36;1mGetting source from Git repository[0;m[0;m
[32;1mGitaly correlation ID: 01K430TNCB31JTNBJNRWEN1ZSD[0;m
[32;1mFetching changes with git depth set to 20...[0;m
Initialized empty Git repository in /builds/root/devsecops/.git/
[32;1mCreated fresh repository.[0;m
fatal: unable to access 'http://127.0.0.1:8099/root/devsecops.git/': Failed to connect to 127.0.0.1 port 8099 after 0 ms: Could not connect to server
section_end:1756743204:get_sources[0K[32;1mRetrying in 5s[0;m
section_start:1756743209:cleanup_file_variables[0K[0K[36;1mCleaning up project directory and file based variables[0;m[0;m
section_end:1756743209:cleanup_file_variables[0K[31;1mERROR: Job failed: exit code 1
[0;m
